app/uk/gov/hmrc/residencenilratebandcalculator/Constants.scala

```diff
@@ -37,6 +39,7 @@ object Constants {
     broughtForwardAllowanceId -> "broughtForwardAllowance",
     chargeableTransferAmountId -> "chargeableTransferAmount",
     dateOfDeathId -> "dateOfDeath",
+    dateOfDisposalId -> "dateOfDisposal",
```

Comment:
We don't want to add this to the jsonKeys yet, as we will be transforming it before sending down the wire, so please remove this for now.